syslog-ng/syslog-ng.conf

```diff
@@ -31,12 +31,19 @@ source s_network {
 };
```

Comment:
Could you please tidy up the patches? The default-network-drivers is removed in the first patch, but re-added in the last.